app/views/hiring_staff/organisations/_vacancies_awaiting_feedback.html.haml

```diff
@@ -1,4 +1,4 @@
-%section.govuk-tabs__panel
+%section.govuk-tabs__panel#vacancies-hits
```

Comment:
wouldn't it be better to put this line outside of all these templates in the presumably one parent template so all the duplication here is removed?